mocks/mocks.py

```diff
@@ -140,6 +142,7 @@ def parse_config(config_file='config_example.yaml',
     # Redshift
     config['redshift'] = kwargs.get('redshift',
                                     config['redshift'])
+    config['cosmology'] = mocks.create_cosmology(config)
```

Comment:
I am not sure about using the function `create_cosmology` inside the function `parse_config`. Any advice about that?